lib/_http_incoming.js

```diff
@@ -137,130 +137,101 @@ function _addHeaderLines(headers, n) {
 
 // 'array' header list is taken from:
 // https://mxr.mozilla.org/mozilla/source/netwerk/protocol/http/src/nsHttpHeaderArray.cpp
```

Comment:
I think this comment might be outdated now? The link is dead at any rate.

Reply:
This link is redirected to [here](https://dxr.mozilla.org/mozilla/source/netwerk/protocol/http/src/nsHttpHeaderArray.cpp). But I think it might not be a good way to keep an unstable url in comment. Should I remove it?

Reply:
I'd be okay with removing it.